README.md

```diff
@@ -110,6 +110,34 @@ module.exports = [{
 }]
 ```
 
+#### `link` hooks
+If you're mastering your plugin which requires some user input (for example, google analytics token), you can use rnpm's prelink and postlink hooks. For instance:
+
+```json
+"rnpm": {
+  "hooks": {
+    "prelink": "./bin/input.js"
+  }
+}
+```
+
+Let's assume we have a `bin/input.js` executable:
+
+```javascript
+const inquirer = require('inquirer');
+
+inquirer.prompt([{
+  message: 'Please, insert here your GA token',
+  name: 'GAToken'
+}], (answers) => {
+  // Do something with answers.GAToken
```

Comment:
Where I write about "if you want to store settings..."? I agree, that this part can be improved, Ill take a look.

Theoretically it can be done this way, but I'd say that there's no way to pass a postlink result to the function body... But, for example, I'd read GA code after installation and make a script to store in the right places.